### ISSUE TITLE
Run performance testing for 1.1 release

### ISSUE THREAD

**Issue by awrichar** [hyperledger/firefly#989]
Child of https://github.com/hyperledger/firefly/issues/563

Need to test the latest code prior to doing a 1.1 release and check for bugs/performance bottlenecks.
Should test with both firefly-ethconnect and with the new firefly-evmconnect.

**Comment by awrichar** [comment]
#563 updated with the results of the latest (~4 day) performance test run. No major issues at this time - will test at least once more once a release candidate is selected.

**Comment by peterbroadhurst** [comment]
Link to the detailed Discord thread: https://discord.com/channels/905194001349627914/1004496575059476570/1020402585628319845

**Comment by peterbroadhurst** [comment]
The summary observations from the final run for 1.1 were:

- We are now spending a lot of time processing receipts, and updating operations, with how chatty EVMConnect is. Large payloads, more often. Optimizing this would be best achieved by switching the webhooks from EVMconnect to just be status changes, and making the large payload be queried through on demand (as we made changes like this on listeners for example).
- The batch assembly loop is running hot too. Uploading blobs is the highest user, followed by flushNonceState.
- The overhead of the eventLoop is also high. We've discussed this many times, and I'm not going to add more here.
- Verbose debug logging is a non-trivial amount of the overall usage at this point. 
Think we can close the 1.1 activities here. No outstanding bugs. Room for optimization as the community decides going forwards, based on specific use cases and functions in projects.
